knotx-knot/knotx-knot-handlebars/src/main/java/io/knotx/knot/templating/impl/HandlebarsFragment.java

```diff
@@ -40,7 +37,7 @@
 
   HandlebarsFragment(Fragment fragment) {
     this.fragment = fragment;
-    this.unwrappedContent = getUnwrappedContent(fragment);
+    this.unwrappedContent = FragmentContentExtractor.getUnwrappedContent(fragment);
```

Comment:
I expect the same change in io.knotx.knot.assembler.impl.UnprocessedFragmentStrategy#UNWRAP.